scripts/supermuc-ng/bashrc_supermuc_ng

```diff
@@ -0,0 +1,13 @@
+module load slurm_setup
```

Comment:
One minor change here: I would name the file just bashrc because Supermuc-NG is already in the name of the directory

Reply:
`bashrc`or `.bashrc`?

Reply:
`bashrc` is better in my opinion, otherwise the user might not see/find the file. One could also call it `bashrc_add_ons` to highlight that this should be added to the `~/.bashrc`

Reply:
I like this idea `bashrc_add_ons`!

Reply:
Done in commit [a857741](https://github.com/exadg/exadg/pull/56/commits/a857741a23912395cd56a0c62d7659a2dffd3c91)